core/src/main/java/com/netflix/msl/msg/MslControl.java

```diff
@@ -834,12 +834,12 @@ public MslControl(final int numThreads, final MessageStreamFactory streamFactory
     }
 
     /**
-     * Assigns a filter stream factory that will be used to filter any incoming
+     * Assigns a filter message factory that will be used to filter any incoming
      * or outgoing messages. The filters will be placed between the MSL message
      * and MSL control, meaning they will see the actual MSL message data as it
      * is being read from or written to the remote entity.
      *
-     * @param factory filter stream factory. May be null.
+     * @param factory filter message factory. May be null.
      */
     public void setFilterFactory(final FilterStreamFactory factory) {
```

Comment:
factory is type is FilterStreamFactory, but comment was changed to say "filter message factory". Should you also rename the FilterStreamFactory Type?